src/app/features/psbt-signer/components/psbt-request-details/components/psbt-inputs-outputs-totals/components/psbt-address-totals.tsx

```diff
@@ -0,0 +1,67 @@
+import { truncateMiddle } from '@stacks/ui-utils';
+
+import { Money } from '@shared/models/money.model';
+
+import { formatMoney, i18nFormatCurrency } from '@app/common/money/format-money';
+import { PsbtInput } from '@app/features/psbt-signer/hooks/use-parsed-inputs';
+import { useCalculateBitcoinFiatValue } from '@app/query/common/market-data/market-data.hooks';
+
+import { PsbtAddressTotalItem } from './psbt-address-total-item';
+import { PsbtInscription } from './psbt-inscription';
+
+interface PsbtAddressTotalsProps {
+  accountInputsTransferringInscriptions?: PsbtInput[];
+  accountOutputsReceivingInscriptions?: PsbtInput[];
+  addressNativeSegwit: string;
+  addressTaproot: string;
+  addressNativeSegwitTotal: Money;
+  addressTaprootTotal: Money;
+  showNativeSegwitTotal: boolean;
+  showTaprootTotal: boolean;
+}
+export function PsbtAddressTotals({
+  accountInputsTransferringInscriptions,
+  accountOutputsReceivingInscriptions,
+  addressNativeSegwit,
+  addressTaproot,
+  addressNativeSegwitTotal,
+  addressTaprootTotal,
+  showNativeSegwitTotal,
+  showTaprootTotal,
+}: PsbtAddressTotalsProps) {
+  const calculateBitcoinFiatValue = useCalculateBitcoinFiatValue();
+
+  const accountHasInputInscriptions = accountInputsTransferringInscriptions?.length;
+  const accountHasOutputInscriptions = accountOutputsReceivingInscriptions?.length;
+
+  return (
+    <>
```

Comment:
Maybe we can use `&&` here and avoid the `?  <> : null` 🤔

Reply:
I do think in our code we tend to pref either an early return or the ternary `null` over `&&` when rendering components. @kyranjamie am I wrong here?

Reply:
Prefer early return where possible. Don't really have a pref on ternary or &&. Would it be worth standardising?